### PR TITLE
add YKZ space

### PR DESCRIPTION
**Pull request by dmitry-morgachev**
Add space for YKZ
  
Etherscan: https://etherscan.io/address/0x87047986e8e4961c11d2edcd94285e3a1331d97b
CoinGecko: https://www.coingecko.com/en/coins/yakuza-dfo
Website: https://www.ykz.network/
Twitter: https://twitter.com/yakuza_dfo

### CODE REVIEW

File: spaces/yakuza/skin.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.yakuza {
+    --primary-color: #d92533;
+    --bg-color: linear-gradient(-45deg,rgba(103,17,41,1) 50%,rgba(26,1,38,1) 100%);
+    --text-color: #aaa;
+    --heading-color: #d92533;
+    --border-color: rgba(0,0,0,0.5);
+    --header-bg: linear-gradient(-45deg,rgba(103,17,41,1) 50%,rgba(26,1,38,1) 100%);
+}
```
Comment (e-asia) [inline spaces/yakuza/skin.scss:8]: The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.

image

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/yakuza/skin.scss (hunk 1 above)
Comment (dmitry-morgachev) [reply spaces/yakuza/skin.scss:8]: @e-asia thank you, reviewed skin. Should be better now.

File: spaces/yakuza/skin.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.yakuza {
+    --primary-color: #d92533;
+    --bg-color: rgb(85,13,40);
+    --text-color: #AAAAAA;
+    --heading-color: #d92533;
+    --border-color: rgba(26,1,38,1);
+    --header-bg: rgb(85,13,40);
+}
```
Comment (e-asia) [inline spaces/yakuza/skin.scss:8]: You still have the Publish button unreadable. 
image

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/yakuza/skin.scss (hunk 2 above)
Comment (dmitry-morgachev) [reply spaces/yakuza/skin.scss:9]: Missed link-color in skin, fixed

**Review by e-asia — APPROVED** [review]

**Pull request merged**